internal/auth/org/org.go

```diff
@@ -0,0 +1,218 @@
+package org
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	ihttp "github.com/llm-operator/cli/internal/http"
+	"github.com/llm-operator/cli/internal/runtime"
+	"github.com/llm-operator/cli/internal/ui"
+	uv1 "github.com/llm-operator/user-manager/api/v1"
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+)
+
+const (
+	path = "/organizations"
+)
+
+// Cmd is the root command for organizations.
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                "organizations",
+		Short:              "organizations commands",
+		Aliases:            []string{"orgs", "org"},
+		Args:               cobra.NoArgs,
+		DisableFlagParsing: false,
+	}
+	cmd.AddCommand(createCmd())
+	cmd.AddCommand(listCmd())
+	cmd.AddCommand(deleteCmd())
+	cmd.AddCommand(addMemberCmd())
+	return cmd
+}
+
+func createCmd() *cobra.Command {
+	var title string
+	cmd := &cobra.Command{
+		Use:  "create",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return create(cmd.Context(), title)
+		},
+	}
+	cmd.Flags().StringVar(&title, "title", "", "Title of the organization")
+	_ = cmd.MarkFlagRequired("title")
+	return cmd
+}
+
+func listCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:  "list",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return list(cmd.Context())
+		},
+	}
+}
+
+func deleteCmd() *cobra.Command {
+	var title string
+	cmd := &cobra.Command{
+		Use:  "delete",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return delete(cmd.Context(), title)
+		},
+	}
+	cmd.Flags().StringVar(&title, "title", "", "Title of the organization")
+	_ = cmd.MarkFlagRequired("title")
+	return cmd
+}
+
+func addMemberCmd() *cobra.Command {
+	var title, email, role string
+	cmd := &cobra.Command{
+		Use:  "add-member",
```

Comment:
Maybe `add-user`? No strong opinion, though.

Reply:
I guess `add-member` is better when we want to add a list command (`list-users` v.s. `list-members`).